src/main/java/net/greatstart/controllers/LoginController.java

```diff
@@ -0,0 +1,28 @@
+package net.greatstart.controllers;
+
+import net.greatstart.dto.DtoUser;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping("/user")
+public class LoginController {
+    @GetMapping(value = "/login")
+    public ModelAndView showLoginForm(String error, String logout) {
+        ModelAndView model = new ModelAndView("login/login");
+        model.addObject(new DtoUser());
+        // TODO localization
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)